packages/info2man.rb

```diff
@@ -0,0 +1,26 @@
+require 'package'
+
+class Info2man < Package
+  description 'Convert GNU info files to POD or man pages.'
+  homepage 'https://www.cskk.ezoshosting.com/cs/css/info2pod.html'
+  version '3.9.8'
+  license 'unknown'
+  compatibility 'all'
+  source_url 'https://salsa.debian.org/debian/info2man.git'
+  git_hashtag 'debian/1.1-10'
+
+  def self.patch
+    system "sed -i 's:#!/usr/bin/perl:#!#{CREW_PREFIX}/bin/perl:g' 0README.txt info2pod"
```

Comment:
Should there be a `0` in front of `README.txt`?

Reply:
If there is, it would make sense to rename to `README.txt`.

Reply:
The README.txt isn't installed, it's just used in the building of the package. I wouldn't worry about it.

Reply:
Then you shouldn't worry about correcting it either. :)